hotsos/core/plugins/storage/ceph.py

```diff
@@ -726,6 +726,14 @@ def osds_pgs_suboptimal(self):
 
         return sorted_dict(_osds_pgs, key=lambda e: e[1], reverse=True)
 
+    @cached_property
```

Comment:
You could use https://github.com/canonical/hotsos/blob/main/hotsos/core/host_helpers/filestat.py to do this. You would need to extend FileObj() with a size property to get the same functionality but then it is nicely re-usable. For an example on how to call it from the scenario yaml see https://github.com/canonical/hotsos/blob/a7f8b1cb31b2bbaf80f73d2305be40a059cf5a26/hotsos/defs/scenarios/openvswitch/ovn/ovn_chassis_ssl_certs.yaml#L2